src/utils/domain/user.ts

```diff
@@ -0,0 +1,31 @@
+import {
+    isNotDefined,
+    isTruthyString,
+} from '@togglecorp/fujs';
+
+interface User {
+    first_name: string | undefined | null;
+    last_name: string | undefined | null;
+    username: string;
+}
```

Comment:
I think we can take it from server. If the server doesn't have the type right now, add a TODO here.